src/systemtest/resources/eiffel-intelligence.sh

```diff
@@ -77,30 +96,32 @@ function usage {
 Usage: systemtest.sh COMMAND [OPTIONS]
```

Comment:
You renamed the script to "eiffel-intelligence.sh ",,, systemtest.sh is not correct.

Reply:
I'll fix this in upcoming changes